server/models/Notification.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const User = require("./User").Schema;
+const userSchema = require("./User");
+const notificationSchema = new mongoose.Schema({
+  type: {
+    type: String,
+    required: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+  },
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  read: {
+    type: Boolean, 
+    default: false
+  },
+  date: {
```

Comment:
Optional: Mongoose schemas have a timestamps option that tells Mongoose to automatically manage `createdAt`

Ref: https://masteringjs.io/tutorials/mongoose/timestamps